climpred/metrics.py

```diff
@@ -1749,25 +1713,26 @@ def _brier_score(forecast, verif, dim=None, **metric_kwargs):
         +-----------------+-----------+
 
     Reference:
-        * Brier, Glenn W. Verification of forecasts expressed in terms of
-          probability.” Monthly Weather Review 78, no. 1 (1950).
-          https://doi.org/10.1175/1520-0493(1950)078<0001:VOFEIT>2.0.CO;2.
         * https://www.nws.noaa.gov/oh/rfcdev/docs/
           Glossary_Forecast_Verification_Metrics.pdf
+        * https://en.wikipedia.org/wiki/Brier_score
 
     See also:
         * properscoring.brier_score
         * xskillscore.brier_score
 
     Example:
-        >>> def pos(x): return x > 0
-        >>> compute_perfect_model(ds, control, metric='brier_score', logical=pos)
+        >>> def pos(x): return x > 0  # checking binary outcomes
+        >>> hindcast.verify(metric='brier_score', comparison='m2o', \
+                dim='member', alignment='same_verifs', logical=pos)
+        >>> hindcast.map(pos).mean('member').verify(metric='brier_score', \
+                comparison='e2o', dim=[], alignment='same_verifs')
     """
     forecast, verif, metric_kwargs, dim = _extract_and_apply_logical(
         forecast, verif, metric_kwargs, dim
     )
```

Comment:
Maybe add a comment here to future developers that `properscoring`/`xskillscore` handles the check that the input datasets contain values from `{0, 1, NaN}`.

Reply:
but this check is not inplace yet everywhere

Reply:
`properscoring` checks this in their base function. So it should error with non-binary numbers, and that error would be thrown from `properscoring`.

Reply:
for brier_score I didnt get a proper error, but some obscure `- not allowed use ~`

Reply:
So maybe we need to replicate this check at some point. Probably in `xskillscore`.